Source/PeleLM.cpp

```diff
@@ -7355,6 +7354,7 @@ PeleLM::compute_Wbar_fluxes(Real time,
                }
                amrex::Real Wbar;
                EOS::Y2WBAR(y, Wbar);
+               Wbar *= 0.001;
```

Comment:
It used to be that Wbar in cgs was in g/mol and in mks was kg/kmol.  So, it must be that somewhere else in all this there is a mol that used to be a kmol that makes all this now correct.  It would be good to understand this (my guess is that it is in the units for R somewhere).